modules/core-functions/pages/base.adoc

```diff
@@ -16,7 +16,8 @@ Initiate a connection object.
 protocol (http:// or https://). If `certPath` is `None` and the protocol is https,
 a self-signed certificate will be used.
 * `graphname`: The default graph for running queries.
-* `gsqlSecret`: The secret key for GSQL. Created in AdminPortal of GraphStudio for the graph you wish to use.
+* `gsqlSecret`: The secret key for GSQL.  +
+See the https://docs.tigergraph.com/tigergraph-server/current/user-access/managing-credentials#_secrets.[documentation] for more details.
```

Comment:
Let's not say "See the documentation for more details". This description is in the documentation - so it's weird to tell users to see the documentation when they are already reading the documentation. We should just use the name of the page or the name of the section.

See Secrets for more details | See Managing Credentials for more details.

Reply:
I agree with this, but the `[documentation] for more details.` is being generated by the doc generation script for all links in the code comments. I am not sure if we can reliably parse the page urls to get something more specific to be generated here.

Reply:
Instead of having the description links generated, maybe we can just write them manually? I don't think we really have that many links in the comments, so i feel like it shouldn't be much of a burden. The description links are just texts like other texts in the description after all